recipes/sol2/config.yml

```diff
@@ -1,4 +1,5 @@
 versions:
+  2.20.6:
+    folder: 2.x.x
   3.0.3:
     folder: 3.x.x
```

Comment:
```suggestion
versions:
  "2.20.6":
    folder: "2.x.x"
  "3.0.3":
    folder: "3.x.x"
```

Reply:
Try passing them to strings. I remember in the past we had a similar problem, that's why we use string in most of config.yml

Reply:
PROJ recipe uses strings and has the same issue.